### PR TITLE
Module4

### PR DESCRIPTION
**Pull request by I-Georg**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: Vagrantfile
Comment (JackMeadDev) [inline Vagrantfile]: It would be much easier to read this without all of the default cruft comments that are no longer required so let's get rid of them, we can always recover them with git or by looking at the vagrant docs

File: Vagrantfile
Comment (JackMeadDev) [inline Vagrantfile:29]: Here you've started one command (adding the `if`) but then interrupt it to do pyenv stuff before finishing off with the `then` - not sure if that's just a typo, but that causes some strange behaviour - now every fresh shell session tries to install python again

File: Vagrantfile
Comment (JackMeadDev) [inline Vagrantfile]: We shouldn't need these steps here are your provisioning script should instead be doing that, just at the moment the step that installs poetry has become hidden inside another statement so isn't being executed

File: README.md
Comment (JackMeadDev) [inline README.md:85]: It would be nice to briefly mention that the option to run your app inside a VM is available, and just to specify any dependencies and how that can be run

File: Vagrantfile
Comment (JackMeadDev) [inline Vagrantfile]: This line isn't doing anything, as the up trigger starts a fresh shell session anyway

**Review by JackMeadDev — APPROVED** [review]
I was able to easily run your VM and access your application, but some of your setup looks a bit confused. I've left a few thoughts below on how we could make that clearer, but I'm happy for you to merge this when you're ready - I'd suggest holding off until Module 3 has also been approved to make it easier to track this

File: Vagrantfile
Comment (I-Georg) [reply Vagrantfile]: Updated, thank you

File: Vagrantfile
Comment (I-Georg) [reply Vagrantfile:29]: This is to have pyenv. Without it being inside pyenv fails

File: Vagrantfile
Comment (I-Georg) [reply Vagrantfile]: Removed,thank you

File: README.md
Comment (I-Georg) [reply README.md:85]: Updated, thank you

File: Vagrantfile
Comment (I-Georg) [reply Vagrantfile]: Updated, thank you

**Pull request merged**
